src/CMakeLists.txt

```diff
@@ -415,6 +437,8 @@ set(libcommon_files
   common/crc32c.cc
   common/crc32c_intel_baseline.c
   common/crc32c_intel_fast.c
+  common/crc32c_ppc.c
+  ${ppc_asm_srcs}
```

Comment:
We can do that, but were trying to follow the example of the Intel assembly ${yasm_srcs} in where and how the list is built and added in.  Which way is preferred?